docs/servo_power_monitor/src/graph.ts

```diff
@@ -35,6 +48,45 @@ export class Graph {
       },
       false
     );
+    if (this.annotationFlag) {
+      this.addAnnotation(
+        powerData[powerData.length - 1][0],
+        this.annotationText
+      );
+      this.annotationFlag = false;
+    }
+  }
+  public setAnnotationFlag(text: AnnotationText) {
```

Comment:
I think this logic (which delays adding an annotation on the next update) should be outside of graph.ts, and the graph only provides a way to set annotation on the last data point. Let's discuss later.